packages/components/src/components/radio-button-wrapper/radio-button-wrapper.tsx

```diff
@@ -63,6 +64,10 @@ export class RadioButtonWrapper {
     this.observeAttributes(); // once initially
   }
 
+  public componentShouldUpdate(newVal: unknown, oldVal: unknown): boolean {
+    return !isDeepEqual(newVal, oldVal);
```

Comment:
Perhaps we should rename this to `hasPropValueChanged` after all? Might be more natural to read than the inversion of this utility.